docs/conf.py

```diff
@@ -42,7 +42,14 @@ def insert_examples():
         f.write("\n".join(lines).encode())
 
 
+def other_doc_tweaks():
+    """Tweak some docstrings to make them more suited for the html docs."""
+    m = jupyter_rfb.RemoteFrameBuffer.handle_event
+    m.__doc__ = m.__doc__.replace("the docs of ``jupyter_rfb.events``", "below")
```

Comment:
I've never seen someone use this hack before. Why not link to the document or whatever it is directly with something like:

```rst
the docs of :mod:`jupyter_rfb.events`.
```

That should link to the module API docs as you expect. I personally hate when docs reference API things and don't link to them in the sphinx docs or assume some other structure (it ends up not being accurate as things change). 75% of my contributions to the dask library are just sphinx docs updates like this.

Reply:
Agreed. I can probably also turn some other references in the guide into links like this.